docs/learning/backend/Fonts.md

```diff
@@ -1,52 +1,86 @@
 
-RSC Fonts are delivered in MicroStation in a resource file known as a Font Library. All of the RSC fonts delivered with MicroStation have been converted to a Font Workspace called `RscFonts.itwin-workspace` that is delivered with iTwin.js. Some user organizations have created their own RSC fonts, and the DGNV8 converter will embed those fonts into iModels created from DGN files that reference them.
+### Embedding rights
 
-### AutoCAD SHX Fonts
+Fonts are generally copyrighted and licensed works. Some may be made available under very permissive licenses allowing them to be shared freely, while others may be subject to very specific rules. iModel authors are responsible for ensuring that their usage of a font is consistent with its licensing terms.
 
-SHX fonts are stored in files, one per font, with an extension ".shx". They may be imported into Font Workspaces by supplying the name of the file.
+The OpenType specification enumerates a handful of possible [embedding licesnging rights](https://learn.microsoft.com/en-us/typography/opentype/spec/os2#fstype) defining whether and for what purposes a font face can be embedded into and distributed with another document like an iModel. [FontFile.isEmbeddable]($backend) will tell you whether embedding is permitted according to these rules. An attempt to embed a non-embeddable font in an iModel will fail.
 
-### TrueType Fonts
+The unit of embedding is a *font file*. Extracting individual font faces from a font file to embed separately is forbidden by most non-free font licenses. The embedding rights for a font file are determined by the most restrictive embedding rights applied to any face within it.
 
-Fonts defined in the `OpenType` format are delivered in files with the extensions: ".ttf", ".ttc", ".otf", or ".otc". They may be imported into Font Workspaces by supplying the name of the  file.
+## Font families
 
-> Note: iTwin.js uses the term `TrueType` where a more accurate term would be `OpenType`. Ignore the distinction.
+A font family is uniquely identified across iModels by its name and type, as encapsulated by [FontFamilyDescriptor]($common). It is possible to have two font families with the same name but of different [FontType]($common)s. Within an iModel, a font family is identified by its unique integer [FontId]($common).
 
-#### TrueType Font Faces
+To use a font family, you must first embed one or more [font files](#font-files) containing the face(s) belonging to that family. Then, you must associate a [FontId]($common) with the [FontFamilyDescriptor]($common). The mapping of family descriptors to font Ids is stored independently from the embedded font files. So, for a given font family, any of the following is possible:
+1. The iModel contains a font Id for the family, and one or more font files containing faces belonging to that family; or
+2. The iModel contains a font Id for the family, but not corresponding font files; or
```

Comment:
I believe that "_not_ corresponding" is intended to be "_no_ corresponding" (which is the wording used in the next item in the list).

Reply:
Please use the suggestion feature for trivial stuff like this.
![image](https://github.com/user-attachments/assets/125da99f-297d-413b-b68f-0f2ba92d8890)

Reply:
```suggestion
2. The iModel contains a font Id for the family, but no corresponding font files; or
```